### PR TITLE
FLT-1816 Setup hosted dev env for web-fleet

### PR DESCRIPTION
**Pull request by kawaic**
- Added serving server side with express
- Updated ci to include static storybook
- Added config selection from environment variable

### CODE REVIEW

**Comment by kawaic** [comment]
Fixed lint errors, added linting to ci  Thank you

File: server.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const path = require('path');
+const app = express();
+const port = 3000;
+
+// heatlh-check should be accessible without basic auth!
```
Comment (abm-trov) [inline server.js]:
```suggestion
// health-check should be accessible without basic auth!
```

File: src/config/develop.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "baseUri": "http://fleet-us-dev-plaformfleet.macronlabs.com:8000",
```
Comment (abm-trov) [inline src/config/develop.json]: Is it `plaformfleet` or `platformfleet`?

File: src/config/staging.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+  "baseUri": "http://fleet-us-staging-plaformfleet.macronlabs.com:8000",
```
Comment (abm-trov) [inline src/config/staging.json]: Same question: is it "plaformfleet" or "platformfleet"?

File: server.js (hunk 1 above)
Comment (kawaic) [reply server.js]: fixed in next PR

File: src/config/develop.json (hunk 2 above)
Comment (kawaic) [reply src/config/develop.json]: fixed in next PR

File: src/config/staging.json (hunk 3 above)
Comment (kawaic) [reply src/config/staging.json]: fixed in next PR

**Pull request merged**